src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -44,7 +44,7 @@ public void init() throws IOException, SaveException {
         Random randomizer = new Random();
         for (int i = 0; i < 1000; i++) {
             BibEntry entry = new BibEntry();
-            entry.setCiteKey("id" + i);
+            entry.setBibTeXKey("id" + i);
```

Comment:
I wanted to be explicit to avoid confusion, because someone might think that a key is an id. And keys are used in SQL-based databases, too. "primary key", "foreign key", etc.